test/e2e/main-e2e.test.js

```diff
@@ -60,4 +61,15 @@ describe('application launch', function () {
     await client.pause(500);
     await client.getWindowCount().should.eventually.equal(initialWindowCount + 1);
   });
+
+  it('check that WebDriverAgent private headers folder (regression test for https://github.com/appium/appium-desktop/issues/417)', async function () {
+    // NOTE: This isn't really an "e2e" test, but the test has to be written here because the /release 
+    // folder needs to be built in order to run the test
+    if (platform === 'darwin') {
```

Comment:
isn't this logic backwards?

Reply:
Yes. @dpgraham just copied-and-pasted my copy-and-paste example.